packages/octavius/octavius.1.2.1/opam

```diff
@@ -0,0 +1,27 @@
+version: "1.2.1"
+opam-version: "2.0"
+name: "octavius"
+maintainer: "leo@lpw25.net"
+authors: [ "Leo White <leo@lpw25.net>" ]
+homepage: "https://github.com/ocaml-doc/octavius"
+doc: "http://ocaml-doc.github.io/octavius/"
+license: "ISC"
+dev-repo: "git+http://github.com/ocaml-doc/octavius.git"
+bug-reports: "https://github.com/ocaml-doc/octavius/issues"
+tags: ["doc" "ocamldoc" "org:ocaml-doc"]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "ocamlfind" {build}
```

Comment:
```suggestion
```
`octavius` doesn't seem to use ocamlfind at all in its source code

Reply:
It uses ocamlbuild with `-use-ocamlfind`

Reply:
Wait, no it doesn't do that anymore, I forgot that it got changed to jbuilder